src/application-navigation/index.js

```diff
@@ -1,10 +1,23 @@
 import {
-  titleConfigPropType, navigationItemsPropType, extensionItemsPropType, utilityItemsPropType, userConfigPropType,
+  titleConfigPropType,
+  navigationItemsPropType,
+  extensionItemsPropType,
+  utilityItemsPropType,
+  userConfigPropType,
 } from './private/utils/propTypes';
+import ApplicationNavigationActionsContext from './ApplicationNavigationActionsContext';
 import ApplicationNavigation from './ApplicationNavigation';
+import ApplicationNavigationWorkspace from './ApplicationNavigationWorkspace';
+import ApplicationNavigationWorkspaceItem from './ApplicationNavigationWorkspaceItem';
 
 export default ApplicationNavigation;
 export {
-  titleConfigPropType, navigationItemsPropType, extensionItemsPropType, utilityItemsPropType, userConfigPropType,
+  titleConfigPropType,
+  navigationItemsPropType,
+  extensionItemsPropType,
+  utilityItemsPropType,
+  userConfigPropType,
+  ApplicationNavigationActionsContext,
+  ApplicationNavigationWorkspace,
+  ApplicationNavigationWorkspaceItem,
```

Comment:
We should consider accessing these through dot notation off application-navigation itself. We did this in v2 and could do it here as well.
ex:
ApplicationNavigation.Workspace
ApplicationNavigation.Workspace.Item

Reply:
Updated.